src/types/chart/chart.ts

```diff
@@ -57,6 +57,37 @@ export interface DatasetValues {
   readonly data: any[];
 }
 
+export interface DatasetDesign {
+  readonly backgroundColor?: string;
+  readonly yAxisID?: string;
```

Comment:
```suggestion
  readonly yAxisId?: string;
```

Reply:
I used this case to make it the same as in ChartJS dataset config :/ But I can change if you really want

Reply:
Yes, I'd rather have consistent naming for our things